app/models/subnet/ipv6.rb

```diff
@@ -26,6 +26,10 @@ def self.supported_ipam_modes
     [:eui64, :db, :none]
   end
 
+  def self.supports_tftp?
+    false
```

Comment:
They do, but Foreman currently does not support this. IMHO tftp orchestration only makes sense in combination with dhcp orchestration. We need efi support in Foreman first before dhcp orchestration can work. So this is still a lot more work to do.